src/V3Gate.cpp

```diff
@@ -524,6 +524,54 @@ class GateClkDecomp final {
     static void apply(GateGraph& graph) { GateClkDecomp{graph}; }
 };
 
+//######################################################################
+// Iterate over vertices looking for wide expressions to exclude from inline
+
+class GateWideExprExclude final {
+    size_t m_statExcluded = 0;  // Statistic tracking
+
+    void exclude(GateVarVertex* const vVtxp) {
+        if (!vVtxp->varScp()->isWide()) return;
+        if (vVtxp->inEmpty() || !vVtxp->inEdges().frontp() || !vVtxp->inEdges().frontp()->fromp())
```

Comment:
This condition is tipple-redundat.

If `vVtxp->inEmpty()` is true, it's guaranteed that vvtxp->inEdges().frontp() is non-null, and a V3GraphEdge cannot exist in a stable state without both `fromp()` and `top()` being non-null., so `vtxp->inEmpty()` implies the other 2 terms.